src/abieos.cpp

```diff
@@ -132,7 +134,9 @@ extern "C" abieos_bool abieos_set_abi_bin(abieos_context* context, uint64_t cont
         from_bin(def, stream);
         abieos::abi c;
         convert(def, c);
-        context->contracts.insert({name{contract}, std::move(c)});
+        const name key {contract};
+        context->contracts.erase(key);
+        context->contracts.insert({key, std::move(c)});
```

Comment:
Should be able to use:  `context->contracts.insert_or_assign(name{contract}, std::move(c));`

Reply:
Yes, insert_or_assign is better.